app/src/containers/ConnectedSidebar/tests/index.test.tsx

```diff
@@ -99,12 +86,14 @@ describe('components/ConnectedSidebar', () => {
 
     const wrapper = mount(
       <Provider store={store}>
-        <MemoryRouter initialEntries={[{ pathname: `/location`, hash: '', search: '', state: {} }]}>
+        <MemoryRouter
+          initialEntries={[{ pathname: `/admin/location/unit`, hash: '', search: '', state: {} }]}
+        >
           <ConnectedSidebar />
         </MemoryRouter>
       </Provider>
     );
-    expect(wrapper.find('Menu').at(0).prop('children')).toMatchSnapshot();
+    expect(wrapper.text()).toMatchSnapshot('Locations unit');
```

Comment:
@HaseemUlHaq This seems to be a test of the page header instead of the menu

Reply:
@HaseemUlHaq at a high level what should these tests achieve?, what pieces of information do we want them to check on each test cycle, for instance are we only interested in knowing that there are this number of menu items, or whether a certain menuItem maps to a respective link?, how then can we code these objective checks in tests such that we have several small snapshots as opposed to one one huge snapshot.